qa/tasks/vstart_runner.py

```diff
@@ -233,7 +233,7 @@ def _write_stderr(self, err):
         else:
             self.stderr.write(err)
 
-    def wait(self):
+    def wait(self, timeout=None):
```

Comment:
commit b1fb94ce7823c8160c857be999ee941ab2de4432 is also in another PR. Could you please send a separate change?

Reply:
Done - https://github.com/ceph/ceph/pull/56888

Reply:
Please remove this commit then.